bundlesize.config.json

```diff
@@ -4,17 +4,17 @@
       "path": "packages/js-horizontal-slider/dist/umd/index.js",
       "maxSize": "7.5 kB"
     },
-    {
-      "path": "packages/js-recommendations/dist/umd/index.js",
-      "maxSize": "8.75 kB"
-    },
     {
       "path": "packages/react-horizontal-slider/dist/umd/index.js",
       "maxSize": "1.75 kB"
     },
     {
-      "path": "packages/react-recommendations/dist/umd/index.js",
-      "maxSize": "3 kB"
+      "path": "packages/recommendations-js/dist/umd/index.js",
+      "maxSize": "7.25 kB"
+    },
+    {
+      "path": "packages/recommendations-react/dist/umd/index.js",
+      "maxSize": "3.25 kB"
```

Comment:
Good question! It went from 3.0KB to 3.1KB. Looking at the diff, it's likely because Babel transpile code multiple times (because the VDOM package is a new dependency) and those polyfills end up being duplicated in the bundle.

I'm able to counter-balance the cost of the abstraction and get it back down to 3.02KB by using the `@babel/plugin-transform-runtime` plugin in e0f94b0006000527f20a227c3877cd55734b5762. Lmk what you think?